src/common/const.ts

```diff
@@ -121,6 +122,7 @@ export const FIXED_DEVICE_CLASS_ICONS = {
   carbon_monoxide: mdiMoleculeCo,
   current: mdiCurrentAc,
   date: mdiCalendar,
+  distance: mdiSpeedometer,
```

Comment:
Would ruler not be better for distance?
```suggestion
  distance: mdiRuler,
```

Reply:
This is an error :) 
We also discussed using:

![CleanShot 2022-09-28 at 09 40 31](https://user-images.githubusercontent.com/195327/192718614-20b7c1a7-d343-4c2f-8bb5-0f5b2eb97e36.png)

Reply:
I updated the icon to what was intended, but I agree @epenet's suggestion would work too:
![image](https://user-images.githubusercontent.com/14281572/192721178-aa7d2521-37fe-4383-a8bc-619f353e01a6.png)
Or maybe:
![image](https://user-images.githubusercontent.com/14281572/192721216-772cef92-4bed-464a-8814-9785edfc3724.png)